src/checkstyle/import-control.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<!DOCTYPE import-control PUBLIC
+        "-//Puppy Crawl//DTD Import Control 1.2//EN"
+        "http://checkstyle.sourceforge.net/dtds/import_control_1_2.dtd">
+<import-control pkg=".*" regex="true">
+    <allow pkg=".*" regex="true"/>
+    <subpackage name="com.facebook.presto">
+        <subpackage name="sql\.(planner|analyzer)" regex="true">
```

Comment:
This seems a little heavy-handed. There may be legitimate uses for IdentityHashMap, so I don't think we should disallow it.

Also, it's brittle, since it can break if we ever move classes or rename packages.

Let's drop this commit.